challenges/lib/src/main/java/challenges/stacksandqueues/PseudoQueue.java

```diff
@@ -0,0 +1,47 @@
+package challenges.stacksandqueues;
+
+public class PseudoQueue {
```

Comment:
You didn't have to re-write your push and pop methods, pseudoQueue could just extend your Stack.

Reply:
Oh okay, Thank you